configure.ac

```diff
@@ -1135,11 +1136,13 @@ AS_IF([test "x$with_openssl" = "xyes"],[
     SQUID_CHECK_OPENSSL_CONST_SSL_SESSION_CB_ARG
     SQUID_CHECK_OPENSSL_CONST_X509_GET0_SIGNATURE_ARGS
     SQUID_CHECK_OPENSSL_TXTDB
+    AC_PATH_PROG(OPENSSL_BIN, openssl)
```

Comment:
Do we really want to ENABLE_CERT_TESTS (below) when neither openssl nor certool binaries are not found (i.e., neither OPENSSL_BIN nor CERTTOOL_BIN variables are defined)?

Should not we ENABLE_CERT_TESTS (below) when Squid is built with GnuTLS but only openssl binary is found (i.e., OPENSSL_BIN is defined but CERTTOOL_BIN is not)? It is currently _not_ enabled in that case AFAICT. Same for OpenSSL+certool combination.

It feels like AC_PATH_PROG() tests for OPENSSL_BIN and CERTTOOL_BIN should be made independent from the corresponding libraries _and_ ENABLE_CERT_TESTS preconditions adjusted to enable testing in more environments where such testing should work.